src/components/AssetSearch/selectors/selectAndSortAssets/selectAndSortAssets.test.ts

```diff
@@ -6,10 +6,10 @@ import { selectAndSortAssets } from './selectAndSortAssets'
 const store = {
   ...mockStore,
   assets: {
-    [rune.tokenId]: rune,
-    [aapl.tokenId]: aapl,
-    [ethereum.chain]: ethereum,
-    [zero.tokenId]: zero
+    [rune.tokenId as string]: rune,
+    [aapl.tokenId as string]: aapl,
+    [ethereum.chain as string]: ethereum,
+    [zero.tokenId as string]: zero
```

Comment:
better fix is to give the assets in `jest/mock/assets` the `Asset` type and we don't have to do this casting everywhere and the tests will be more robust

Reply:
I did give the assets in `jest/mock/assets` the `Asset` type. Without the casting, tsc complains here about the `.tokenId` fields being of type `string | undefined` when a key of type `string` is required.